packages/itwinui-react/src/core/ExpandableBlock/ExpandableBlock.tsx

```diff
@@ -135,7 +135,7 @@ const ExpandableBlockHeader = React.forwardRef((props, forwardedRef) => {
       as='button'
       type='button'
       className={cx('iui-expandable-header', className)}
-      aria-expanded={isExpanded}
+      aria-expanded={isExpanded ?? !disabled}
```

Comment:
why?

```suggestion
      aria-expanded={isExpanded}
```

Reply:
I was trying to pass this unit test 

```
it('should respect disabled prop', () => {
  const onToggleMock = jest.fn();
  const { getByRole } = render(
    <ExpandableBlock onToggle={onToggleMock} disabled>
      <ExpandableBlock.Header label='test title' />
      <ExpandableBlock.Content>test content</ExpandableBlock.Content>
    </ExpandableBlock>,
  );
  const header = getByRole('button');
  expect(header).toHaveAttribute('aria-disabled', 'true');
  fireEvent.click(header);
  expect(onToggleMock).not.toHaveBeenCalled();
  expect(header).toHaveAttribute('aria-expanded', 'false');
});

```

Since passing `disabled` only adds `aria-disabled` attribute and not `aria-expanded`, I try to explicitly add it this way when `isExpanded` is not present

Reply:
discussed offline and fixed in [`65ba13b`](https://github.com/iTwin/iTwinUI/pull/1354/commits/65ba13b0dc9d475ec2a034b264f83186e7aa0362)